app/views/shared/check_answers/_employed_income.html.erb

```diff
@@ -1,14 +1,16 @@
 <% if partner
      individual = "partner"
+     individual_with_determiner = "the partner"
      model = @legal_aid_application.partner
      url = providers_legal_aid_application_partners_employment_income_path(@legal_aid_application)
    else
      individual = "client"
+     individual_with_determiner = "your partner"
```

Comment:
translations needed?

Reply:
Yep, this was actually incorrect too, it should be 'your client' 🙈 updated!